includes/ingester.php

```diff
@@ -11,7 +11,7 @@ function research_briefings_wp_read_research_briefings() {
 		return false;
 	}
 
-	$research_briefings = file_get_contents('https://lda.data.parliament.uk/researchbriefings.json?_pageSize=20');
+	$research_briefings = file_get_contents('https://lda.data.parliament.uk/researchbriefings.json?_pageSize=500');
```

Comment:
I'm slightly conscious of requesting 500 research briefings every hour. Can we drop this back down to 20?

Reply:
Hi Jake. This was still 500 on the live site, so I am happy to change it to 20 if that is what it should be now.

Reply:
Please - it was originally 500 to update previous briefing papers.